input/postgres/statements.go

```diff
@@ -234,6 +235,20 @@ func GetStatements(server *state.Server, logger *util.Logger, db *sql.DB, global
 		if showtext {
 			statementTexts[key] = receivedQuery.String
 		}
+		if queryID.Valid && showtext {
+			if server.PrevState.QueryIdentities == nil {
+				server.PrevState.QueryIdentities = make(state.QueryIdentityMap)
+			}
+			if identity, ok := server.PrevState.QueryIdentities[queryID.Int64]; ok {
+				identity.LastSeen = time.Now()
+			} else {
```

Comment:
It may be worth caching `time.Now()` at the beginning of the function, or using the collected at time - getting the time ~5000 times (or more, depending on query count and pg_stat_statements.max) isn't free, I think.